src/main/java/org/zalando/nakadi/domain/ResourceImpl.java

```diff
@@ -17,6 +18,7 @@
     private final T resource;
     private final String name;
     private final String type;
+    @Nullable
```

Comment:
It doesn't make much sense to mark private values as `@Nullable`. It's better to mark it nullable in public methods (e.g. constructor parameter, getter)

Reply:
Fixed this. :sweat_smile: